packages/components/src/button/src/Button.css

```diff
@@ -212,19 +257,34 @@ a.o-ui-button {
 .o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading).o-ui-button-hover {
     background-color: var(--hop-neutral-surface-hover);
     color: var(--hop-neutral-text-hover);
+    text-decoration: underline;
 }
 
 /* TERTIARY | FOCUS */
 .o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading):focus-visible,
 .o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading).o-ui-button-focus {
     background-color: var(--hop-neutral-surface-hover);
     color: var(--hop-neutral-text-hover);
+    text-decoration: underline;
 }
 
 /* TERTIARY | ACTIVE */
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading):active,
 .o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading).o-ui-button-active {
-    background-color: transparent;
-    color: var(--hop-neutral-text);
+    background-color: var(--hop-neutral-surface-press);
+    color: var(--hop-neutral-text-press);
+    text-decoration: underline;
+}
+
+/* TERTIARY | SELECTED / SELECTED + ACTIVE */
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"],
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"].o-ui-button-active,
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"]:active,
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-checked="true"],
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-checked="true"].o-ui-button-active,
+.o-ui-button-tertiary:not(.o-ui-button-inherit-style):not([disabled]):not(.o-ui-button-loading)[aria-checked="true"]:active {
```

Comment:
these selectors are really out of control.. I wish we could do something to improve them. Sometimes you don't need to put :not everywhere as long as those versions override these ones, but it may not be easy in this situation.

Reply:
Always have been like this IIRC, I'd rather use not than redeclare some CSS rules.